firebase-firestore/src/main/java/com/google/firebase/firestore/local/BundledDocumentMetadata.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore.local;
+
+import com.google.firebase.firestore.model.DocumentKey;
+import com.google.firebase.firestore.model.SnapshotVersion;
+import java.util.List;
+
+/** Metadata describing a Firestore document saved in the bundle. */
+public class BundledDocumentMetadata {
```

Comment:
I think I might need this class since I cannot directly access a JSON object, but need to convert JSON in Java before I can use it in the BundleLoader. I added a TODO to see if I can remove it.